packages/pigment-css-react/tests/sx/fixtures/sxProps.output.js

```diff
@@ -1,43 +1,52 @@
-import { styled as _styled } from '@pigment-css/react';
+import { sx as _sx8, styled as _styled } from '@pigment-css/react';
+import { ForwardSx as _ForwardSx4 } from '@pigment-css/react/private-runtime';
+import { sx as _sx6 } from '@pigment-css/react';
+import { ForwardSx as _ForwardSx3 } from '@pigment-css/react/private-runtime';
+import { sx as _sx4 } from '@pigment-css/react';
+import { ForwardSx as _ForwardSx2 } from '@pigment-css/react/private-runtime';
+import { sx as _sx2 } from '@pigment-css/react';
+import { ForwardSx as _ForwardSx } from '@pigment-css/react/private-runtime';
```

Comment:
Why do we have this many duplicated imports? Can you explain how does this work?

Reply:
This is the generated output. For each usage of any of `css/styled/sx`, their processors imports their own required imports as the context is specific for each and is not shared across invocations.
So even if there are two `styled()` calls within the same file, they'll import and use their own identifiers.

Reply:
> This is the generated output.

Yeah, I know, I was just surprised at how it looked like. I wondered if there is not a way to import it once and reuse the same import.

Reply:
If you'd like to add only one import per file, you can probably consider to add state outside of the import statements logic, for example:
```ts
// enum to decrease memory usage in cases w/ a lot of files (number vs string)
enum ImportType {
	forwardSx,
	sx,
	styled
}
type state = {
	[fileId: string]: {
		[importStatement in ImportType]?: true
	}
}
const state: state = {}

const addImportStatement = (fileId: string, importType: ImportType) => {
	if (!state[fileId]) state[fileId] = {}
	if (!state[fileId][importType]) {
		state[fileId][importType] = true
		// ... add import statement
	}
}
```

Reply:
Even if we did this for `Forward` sx, the imports added for `sx` itself can't be updated to do this since the context that `wyw` passes to each processor does not included a reference to the current file's ast. So it'll still continue to be the same for `sx/css/styled` calls.